src/tools/kata-ctl/src/arch/powerpc64le/mod.rs

```diff
@@ -20,4 +21,19 @@ mod arch_specific {
     pub fn get_checks() -> Option<&'static [CheckItem<'static>]> {
         None
     }
+
+    const PEF_SYS_FIRMWARE_DIR: &str = "/sys/firmware/ultravisor/";
+
+    pub fn available_guest_protection() -> Result<check::GuestProtection>{
```

Comment:
Could you run this code through `rustfmt` to give it the standard layout?

```suggestion
    pub fn available_guest_protection() -> Result<check::GuestProtection> {
```